Libraries/NativeAnimation/RCTNativeAnimatedNodesManager.m

```diff
@@ -264,6 +265,20 @@ - (void)stopAnimation:(nonnull NSNumber *)animationId
   }
 }
 
+- (void)stopAnimationsForNode:(nonnull RCTAnimatedNode *)node
+{
```

Comment:
I need to loop through `_activeAnimations` which is of type `NSMutableSet` whereas on android is just an array.
Not sure what is your suggestion on how to use a "normal for loop"? If I call `[_activeAnimations allObjects]` that would allocate new array for all the items anyways.

Complexity wise its O(n) either way unless we have mapping from node to active animations running for that node which IMO is not worth adding considering that the number of running animations should be very low (like literally just a few of them).